### ISSUE TITLE
Dependency issue while installing cloudlift as a python library

### ISSUE THREAD

**Issue by Equlnox** [GetSimpl/cloudlift#27]
As of now, `setup.py` directly imports `VERSION` from `cloudlift.version`.
As a result of that, `cloudlift.__init__` runs and imports dependencies.
This is a circular dependency. To resolve this, we can directly read the version file instead of importing it.

Here is the error stack trace form a dockerized application that I was trying to build.

```Obtaining cloudlift from git+ssh://****@github.com/Rippling/cloudlift.git@master#egg=cloudlift (from -r requirements.txt (line 10))
  Cloning ssh://****@github.com/Rippling/cloudlift.git (to revision master) to ./src/cloudlift
  Running command git clone -q 'ssh://****@github.com/Rippling/cloudlift.git' /container-runner-api/src/cloudlift
  Warning: Permanently added the RSA host key for IP address '13.234.210.38' to the list of known hosts.
    ERROR: Command errored out with exit status 1:
     command: /usr/local/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/container-runner-api/src/cloudlift/setup.py'"'"'; __file__='"'"'/container-runner-api/src/cloudlift/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info
         cwd: /container-runner-api/src/cloudlift/
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/container-runner-api/src/cloudlift/setup.py", line 3, in <module>
        from cloudlift.version import VERSION
      File "/container-runner-api/src/cloudlift/cloudlift/__init__.py", line 3, in <module>
        import boto3
    ModuleNotFoundError: No module named 'boto3'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```
One simple solution would be to exec `version/__init_.py` instead of importing it

inside `setup.py`
```
version = {}
with open("./cloudlift/version/__init__.py") as fp:
    exec(fp.read())
```
Another solution could be creating a seperate version.py file and defining `__version__` globally inside it.

inside `version.py`
```__version__ = "1.3.1"```

inside `setup.py`
```
version = {}
with open("./cloudlift/version.py") as fp:
    exec(fp.read(), version)
# version can be accessed using version["__version__"]
```
Other packages could simply import this and access `__version__`

**Comment by Equlnox** [comment]
#30 Only changes `setup.py` and fixes the import.

**Comment by shoan** [comment]
How about having the version in `cloudlift/__init__.py`?

```
>>> import cloudlift
>>> cloudlift.__version__
'1.4.2'
```

**Comment by siliconsenthil** [comment]
Yeah. we can do that also. But that's what initially we've done in previous PR (that closed). But that means we need to change in multiple places to replace VERSION with _version_ which means we need to test. 

So we've created a PR with surgical change. Nothing more than fixing the bug and easier for you to quickly test.  :) 

We can do the refactoring later as a separate PR if needed

**Comment by Equlnox** [comment]
Fixed in #30
